src/js/stores/__tests__/ChronosStore-test.js

```diff
@@ -35,7 +40,7 @@ describe('ChronosStore', function () {
       jasmine.clock().tick(2 * Config.getRefreshRate());
       // Finish up outstanding timers
       jest.runOnlyPendingTimers();
-      expect(ChronosActions.fetchJobs.calls.count()).toEqual(2);
+      expect(ChronosActions.fetchJobs.calls.count()).toEqual(3);
```

Comment:
This test needs to be changed to 3 calls of `#fetchJobs`. `#fetchJobs` is called immediately when `setInterval` is initiated, then twice more due to the mocked passage of time (`2 * Config.getRefreshRate()`).